frontend/src/services/ApiService.ts

```diff
@@ -24,21 +24,27 @@ export const transformAxiosError = (e: Error): ApiError => {
     if (e.response) {
       const statusCode = e.response.status
       if (statusCode === StatusCodes.TOO_MANY_REQUESTS) {
-        return new HttpError('Please try again later.', statusCode)
+        return new HttpError('Error [429]: Please try again later.', statusCode)
```

Comment:
How about `Error [001]: Please try again later` (and downshifting the rest) so that the numbering is consistent for tracing?

Reply:
same as above